infra/src/main/scala/ch/epfl/scala/utils/DoobieUtils.scala

```diff
@@ -71,4 +70,41 @@ object DoobieUtils {
       buildSelect(table, fields) ++ space ++ where
   }
 
+  object Mappings {
+    implicit val stringMeta: Meta[String] = Meta.StringMeta
+    implicit val URLDecoder: Decoder[Url] = deriveDecoder[Url]
+    implicit val URLEncoder: Encoder[Url] = deriveEncoder[Url]
+    implicit val contributorMeta: Meta[List[GithubContributor]] = {
+      implicit val contributorDecoder: Decoder[GithubContributor] =
+        deriveDecoder[GithubContributor]
+      implicit val contributorEncoder: Encoder[GithubContributor] =
+        deriveEncoder[GithubContributor]
+      stringMeta.timap(
+        _.split(",")
+          .filter(_.nonEmpty)
+          .map(fromJson[GithubContributor](_).get)
+          .toList
+      )(_.map(toJson(_)).mkString(","))
+    }
+    implicit val githubIssuesMeta: Meta[List[GithubIssue]] = {
+      implicit val githubIssueDecoder: Decoder[GithubIssue] =
+        deriveDecoder[GithubIssue]
+      implicit val githubIssueEncoder: Encoder[GithubIssue] =
+        deriveEncoder[GithubIssue]
+      stringMeta.timap(
+        _.split(",").filter(_.nonEmpty).map(fromJson[GithubIssue](_).get).toList
+      )(_.map(toJson(_)).mkString(","))
```

Comment:
Why don't you use circe derivation to create an encoder and a decoder of `List[GithubIssue]`